calendarsubscription/calendarsubscription_test.go

```diff
@@ -0,0 +1,43 @@
+package calendarsubscription
+
+import (
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/target/goalert/oncall"
+)
+
+func TestRenderICalFromShifts(t *testing.T) {
```

Comment:
Since this is a method test, it should be named in the style `TestTypeName_MethodName`
```suggestion
func TestCalendarSubscription_RenderICalFromShifts(t *testing.T) {
```